src/main/java/data/Account.java

```diff
@@ -130,6 +130,23 @@ public void setAggregationMode(String aggregationMode) {
     this.aggregationMode = aggregationMode;
   }
 
+  /** Builds one row filled with an Account's data for the Sheet. */
+  public List getAccountSheetsRow() {
+    String[] accountData = { 
+      getAccountID(), 
+      getVendorID(), 
+      getEntity(), 
+      getCurrency(), 
+      getDirection(),
+      getLegacyAccountID(),
+      "" + getNextGenAccountID(),
+      getMatchingMode(),
```

Comment:
What does the empty string before `getNextGenAccountId()` do?

Reply:
Since this is a String Array, I can only input Strings. getNextGenAccountID() returns an int so the "" in front of it turns the response into a String.

Reply:
Use Integer.toString() for this instead of concatenating a ""